DBugVision/main.py

```diff
@@ -0,0 +1,36 @@
+# Imports:
+
+import constants
+import dbug_networking
+import dbug_video_stream
+
+
+# NOTE: For Barak's QA - is this the right place to write such function?
+def filter_sort_contours(contours):
+    pass
+
+if __name__ == "__main__":
+
+    # TODO: Create a lock file to be sure that no other process is currently processing and sending data to the robot
+
+    # The communication channel to the robot
+    robot_com = dbug_networking.DBugNetworking()
+    # Video stream to get the images from
+    cam = dbug_video_stream.DBugVideoStream(camera_device_index=constants.CAMERA_INDEX)

```

Comment:
Didn't we have issues where this would change from 0 to 1 due to some USB voodoo magic?
Shouldn't there be a 'best effort' style method which tries to connect to _any_ device?